k8s/conf/icmp-responder-nginx.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        networkservicemesh.io/app: "icmp-responder"
+        networkservicemesh.io/impl: "icmp-responder"
+    spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: networkservicemesh.io/app
+                    operator: In
+                    values:
+                      - icmp-responder
+                  - key: networkservicemesh.io/impl
+                    operator: In
+                    values:
+                      - icmp-responder
+              topologyKey: "kubernetes.io/hostname"
+      containers:
+        - name: icmp-responder-nse
+          image: networkservicemesh/icmp-responder-nse:latest
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: NSE_LABELS
+              value: "impl=kernel"
+            - name: IP_ADDRESS
+              value: "10.20.1.1"
```

Comment:
Lets make this "10.60.1.1" just to make sure we are easy to distinguish from other NSEs.